src/workspaces/list/WorkspaceFilters.ts

```diff
@@ -32,14 +32,14 @@ export const WorkspaceFilters = (props: WorkspaceFiltersProps): ReactNode => {
   const { query } = Nav.useRoute();
   const filters = getWorkspaceFiltersFromQuery(query);
 
-  let keywordLastEvented = useInstance(() => filters.nameFilter);
+  let keywordLastEvented = useInstance(() => filters.keywordFilter);
```

Comment:
This was changed to `nameFilter` to make it clearer that it was only used for searching name and namespace, but now we are expanding the usage so I'm changing it back to `keywordFilter` (which matches existing nomenclature used in the metrics event.